tools/performance-server/ui/chartist/scss/chartist.scss

```diff
@@ -0,0 +1,241 @@
+@import "settings/chartist-settings";
```

Comment:
Not sure if we shall commit third-party code like that this way.

Reply:
I had problems with auto installing bower and library as npm module, I had to install it calling install command by myself. I couldn't do this on kubernetes. 

I saved license file for this library.